person.rb

```diff
@@ -1,4 +1,9 @@
 # class representing a person
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
 class Person
```

Comment:
- [ ] It will be great to ensure that the person class inherits from `nameable` class. This is a requirement for this project to be approved

Reply:
Okey Thanks